src/main/java/com/user/db/repository/UserRepository.java

```diff
@@ -0,0 +1,79 @@
+package com.user.db.repository;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+import com.user.db.entity.User;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class UserRepository {
+
+	private final String dbURL = "jdbc:mysql://localhost:3306/user";
+	private final String username = "root";
+	private final String password = "123";
+
+	private final String SQL_SELECT = "SELECT * FROM user WHERE `id` = ?;";
+	private final String SQL_INSERT = "INSERT INTO user (firstname, middlename, lastname) VALUES(?, ?, ?);";
+	private final String SQL_UPDATE = "UPDATE user SET firstname = ?, middlename = ?, lastname = ? WHERE `id` = ?;";
+	private final String SQL_DELETE = "DELETE FROM user WHERE `id` = ?;";
+
+	public User getUser(String id) throws SQLException, ClassNotFoundException {
+		Class.forName("com.mysql.jdbc.Driver");
+		try (Connection myConnection = DriverManager.getConnection(dbURL, username, password)) {
```

Comment:
Do not create a db connection every time. Create a final class variable and initialize it in the class constructor, then reuse it everywhere.